test/topology_custom/test_change_ip_repro.py

```diff
@@ -0,0 +1,46 @@
+import logging
+import time
+import pytest
+
+from test.pylib.manager_client import ManagerClient
+from test.pylib.rest_client import inject_error
+from test.pylib.util import read_barrier, wait_for_cql_and_get_hosts
+
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.mark.asyncio
+async def test_change_ip_repro(manager: ManagerClient) -> None:
```

Comment:
Please change the name of the test (and the file) to be more specific.
For example: `test_old_ip_notification_repro`